### DISCUSSION TITLE
Skipping build for Windows because C:\Users\builder\clone\windows runner does not exist.

### DISCUSSION THREAD

**Discussion by markhorrocks** [codemagic-ci-cd/codemagic-docs#988] [Q&A] [answered]
I am trying to build a flutter app for Windows but it fails with `Skipping build for Windows because C:\Users\builder\clone\windows runner does not exist.`. 

I have developed the app in ubuntu and connected it to codemagic with the Github repo for the project.

**Comment by himesh-cm** [comment, marked as answer]
Hi @markhorrocks can you please share the build id, so we can look into the issue further?

**Reply by markhorrocks** [reply]
The build ID is 63331c1f6e6d53de3f6fef48

**Reply by himesh-cm** [reply]
Hi @markhorrocks, There is not much in the logs. I ran a quick build at my end, and it worked fine. 

Can you please share the information below to investigate the issue further?
1. Does it work locally for you?
2. Did you use `flutter create` to create the windows app? - more info [here](https://github.com/flutter/flutter/issues/33726)
3. Can you run another build with `--verbose` and share the build id? You can do this in your WFE>Build settings

Thank you.

**Reply by markhorrocks** [reply]
1. No, I can't run flutter build windows on linux. 
2. No, I created the app on linux as a mobile app. 
3. I'm totally lost how to do this. A search turned up no documentation.

**Reply by himesh-cm** [reply]
Thanks for the info. Please check the attachment for point no.3
<img width="816" alt="Screenshot 2022-09-28 at 12 26 12 PM" src="https://user-images.githubusercontent.com/106069365/192709537-2cf22c58-7229-4c1b-aaab-4d13ab75b248.png">

Please share the build id once you are done.

**Reply by markhorrocks** [reply]
Verbose build doesn't appear to have additional info - 6333f2fc0c4aebe2e7df2a35

**Reply by himesh-cm** [reply]
Can you please `flutter build windows` in your pre-build script?
You should see a + icon above build settings >> add it under `Pre-build script` and share the new build id.

Thank you.

**Reply by markhorrocks** [reply]
build with `flutter build windows` - 63340a106e6d531af50dc227

**Reply by himesh-cm** [reply]
As the error says 
`
No Windows desktop project is configured. See https://docs.flutter.dev/desktop#add-desktop-support-to-an-existing-flutter-app to learn about adding Windows support to a project.
`

You need to configure your flutter project for windows support before your run any builds on CI.

Please try the above solutions; let us know if you need further support.

**Reply by markhorrocks** [reply]
This now works, thanks.

**Reply by himesh-cm** [reply]
Great. Please mark it as an answer, so others will find it helpful too.
